Sources/Sentry/SentrySDK.m

```diff
@@ -160,10 +160,11 @@ + (void)startWithOptions:(SentryOptions *)options
     SENTRY_LOG_DEBUG(@"Dispatching init work required to run on main thread.");
     [SentryThreadWrapper onMainThread:^{
         SENTRY_LOG_DEBUG(@"SDK main thread init started...");
-        [SentrySDK installIntegrations];
 
         [SentryCrashWrapper.sharedInstance startBinaryImageCache];
         [SentryDependencyContainer.sharedInstance.binaryImageCache start];
+
+        [SentrySDK installIntegrations];
```

Comment:
`h`: I'm assuming this was moved because it now needs the cache. However, we should be **extremely** cautious with changes like these, please double- and triple-check that this doesn't break things

Reply:
TBH this makes more sense than before, we definitely need SentryDependencyContainer ready before installing all integrations since the integrations uses the dependency container.